docs/MultiClusterObservability-CRD.md

```diff
@@ -23,103 +23,184 @@ observability.open-cluster-management.io/v1beta1
    <td><strong>Req’d</strong>
    </td>
   </tr>
+  <tr>
+   <td>EnableDownsampling
+   </td>
+   <td>bool
+   </td>
+   <td>Enable or disable the downsample.
```

Comment:
should we use downsampling or downsample?

Reply:
yes. should be `downsampling`